diff --git a/configure.ac b/configure.ac
index c86f6c9..faea7e5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -536,17 +536,17 @@ fi
 
 AH_TEMPLATE(C_SDL_SOUND,[Define to 1 to enable SDL_sound support])
 AC_CHECK_HEADER(SDL_sound.h,have_SDL_sound_h=yes,)
-AC_CHECK_LIB(SDL_sound, Sound_Init, have_SDL_sound_init=yes,,)
-AC_CHECK_LIB(SDL_sound, Sound_Seek, have_SDL_sound_seek=yes,,)
+AC_CHECK_LIB(SDL2_sound, Sound_Init, have_SDL_sound_init=yes,,)
+AC_CHECK_LIB(SDL2_sound, Sound_Seek, have_SDL_sound_seek=yes,,)
 if test x$have_SDL_sound_h = xyes -a x$have_SDL_sound_init = xyes ; then
   if test x$have_SDL_sound_seek = xyes ; then
-    LIBS="-lSDL_sound $LIBS"
+    LIBS="-lSDL2_sound $LIBS"
     AC_DEFINE(C_SDL_SOUND,1)
    else 
-     AC_MSG_WARN([Can't find SoundSeek in libSDL_Sound, libSDL_sound support disabled])
+     AC_MSG_WARN([Can't find SoundSeek in libSDL2_Sound, libSDL2_sound support disabled])
    fi
 else
-  AC_MSG_WARN([Can't find libSDL_sound, libSDL_sound support disabled])
+  AC_MSG_WARN([Can't find libSDL2_sound, libSDL2_sound support disabled])
 fi
 
 dnl Check for mprotect. Needed for 64 bits linux 
